app/views/reimbursement_requests/_lightbox.html.erb

```diff
@@ -0,0 +1,17 @@
+
+
+<% @reimbursement_request.attachments.each do |child| %>
+
+  <div id="mod", class="attachment-<%= child.id %> modal">
```

Comment:
there shouldn't be a comma in html attributes

Reply:
Also, `id` should be unique in DOM elements.. this loop is generating a variable number of these DIVs so they should not all have the same ID.. in this case, the ID isn't necessary since the class is generated to be dynamic `attachment-#`